### PR TITLE
feat: package timeout for k8s communication

### PR DESCRIPTION
**Pull request by smuu**
Resolves https://github.com/celestiaorg/knuu/issues/115

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: pkg/k8s/k8s.go [hunk 1]
```diff
@@ -18,6 +19,8 @@ var clientset *kubernetes.Clientset
 // namespace is the current namespace in use by the Kubernetes client.
 var namespace = ""
 
+var timeout = 20 * time.Second
```
Comment (jrmanes) [inline pkg/k8s/k8s.go:22]: `const` maybe better?

**Review by jrmanes — DISMISSED** [review]
just a comment, LGTM apart form this one 🚀

File: pkg/k8s/k8s.go (hunk 1 above)
Comment (smuu) [reply pkg/k8s/k8s.go]: That's better, thank you :+1:

**Review by jrmanes — APPROVED** [review]
LGTM 🔥

**Pull request merged**